knora-ontologies/knora-base.ttl

```diff
@@ -2730,11 +2700,7 @@
 
 :TextFileValue rdf:type owl:Class ;
```

Comment:
It would make sense to store it in the triplestore if we could search its contents using full-text search. I think Lucene can be configured to handle XML syntax, but we'd have to look into this. Otherwise, I think it makes more sense to store it in a file.